src/spryker/application/skeleton/composer-harness.json.twig

```diff
@@ -23,7 +23,11 @@
   "autoload-dev": {
```

Comment:
as its in "autoload-dev", wouldn't that only improve local and CI, not pipeline environments?

Reply:
You're right, it won't affect the environment - it's more for local and CI improvements 👍

Reply:
ok to reword my question another way, would pipeline environments see any benefit in also having it? Or is it already in place in the sub packages?

Reply:
I don't think pipeline environments would see a benefit (we haven't measured that on Atida, just that our CI jobs in Gitlab now take significantly less time to run).

Reply:
is the pipeline environment already using classmaps for these packages? If not then we should make it do so for both performance and consistency between environments

Reply:
The full classmap is dumped for pipeline envs as part of the CI build

Reply:
ok thanks, I just checked and definitely can see vendor/Spryker and vendor/SprykerShop in pipeline environments